### ISSUE TITLE
Structure panel shows extraneous "V" entries for call to "func ()"

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#158]
plugin version: 0.74

### Description

The second entry in the structure panel doesn't define anything.  It should not be present.

![image](https://user-images.githubusercontent.com/1253341/55184189-ca02f080-514e-11e9-8eb7-4abfb2bf015a.png)

```
let process_args () =
  Printf.printf ""
  
let () =
    Printf.printf "";
    process_args ()
```

**Comment by giraud** [comment]
Fixed in 0.75

**Comment by jfehrle** [comment]
Good!
